src/registrar/management/commands/transfer_transition_domains_to_domains.py

```diff
@@ -0,0 +1,229 @@
+"""Load domain invitations for existing domains and their contacts."""
+
+import logging
+import argparse
+
+from django_fsm import TransitionNotAllowed  # type: ignore
+
+from django.core.management import BaseCommand
+
+from registrar.models import TransitionDomain
+from registrar.models import Domain
+
+logger = logging.getLogger(__name__)
+
+
+class termColors:
+    """Colors for terminal outputs
+    (makes reading the logs WAY easier)"""
+
+    HEADER = "\033[95m"
+    OKBLUE = "\033[94m"
+    OKCYAN = "\033[96m"
+    OKGREEN = "\033[92m"
+    WARNING = "\033[93m"
+    FAIL = "\033[91m"
+    ENDC = "\033[0m"
+    BOLD = "\033[1m"
+    UNDERLINE = "\033[4m"
+    BackgroundLightYellow = "\033[103m"
+
+
+# ----------------------------------------
+#  MAIN SCRIPT
+# ----------------------------------------
+class Command(BaseCommand):
+    help = """Load data from transition domain tables
+    into main domain tables."""
+
+    def add_arguments(self, parser):
+        parser.add_argument("--debug", action=argparse.BooleanOptionalAction)
+
+        parser.add_argument(
+            "--limitParse", default=0, help="Sets max number of entries to load"
+        )
+
+    def handle(  # noqa: C901
+        self,
+        **options,
```

Comment:
like with the other PR I would ask that this get broken up into other functions to make it a bit more readable.

Reply:
Done